substrate/frame/treasury/src/lib.rs

```diff
@@ -203,9 +203,6 @@ pub mod pallet {
 		/// The staking balance.
 		type Currency: Currency<Self::AccountId> + ReservableCurrency<Self::AccountId>;
 
-		/// Origin from which approvals must come.
-		type ApproveOrigin: EnsureOrigin<Self::RuntimeOrigin>;
```

Comment:
I think there is more parameters that are not relevant anymore, ProposalBond for example

Reply:
Resolved in https://github.com/paritytech/polkadot-sdk/pull/3820/commits/2f1c63220516be9d50dff104bc41a0621e950c33 and https://github.com/paritytech/polkadot-sdk/pull/3820/commits/3d26a9235d84fb3b353ac537e4c1b49d482581a1